packages/asc-ui/src/components/Typography/TypographyStyle.ts

```diff
@@ -31,12 +28,6 @@ export default styled.p<Props>`
   ${getTypographyFromTheme()};
   font-stretch: normal;
   letter-spacing: normal;
-  ${({ strong }) =>
```

Comment:
please revert this. Now we're only able to set strong on paragraph and no other typeography-related components

Reply:
Sure! Thanks!